files/drop-in-css/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "drop-in.css",
+  "repo": "webdevjeffus/drop-in-css",
+  "files": {
+    "basePath": "minimized/"
```

Comment:
There is no such folder here https://github.com/webdevjeffus/drop-in-css/tree/v1.0.0
The bot will fail